app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurance/InsuranceDestination.kt

```diff
@@ -145,59 +152,62 @@ private fun InsuranceScreen(
     refreshingOffset = PullRefreshDefaults.RefreshingOffset + systemBarInsetTopDp,
   )
   Box(Modifier.fillMaxSize()) {
-    AnimatedContent(
-      targetState = uiState.isLoading,
-      transitionSpec = {
-        MotionDefaults.fadeThroughEnter togetherWith MotionDefaults.fadeThroughExit
-      },
-      label = "uiState is Loading",
-      modifier = Modifier.fillMaxSize(),
-    ) { isLoading ->
-      when (isLoading) {
-        true -> HedvigFullScreenCenterAlignedProgressDebounced(Modifier.fillMaxSize())
-        false -> {
-          Column(
-            Modifier
-              .fillMaxSize()
-              .pullRefresh(pullRefreshState)
-              .verticalScroll(rememberScrollState())
-              .windowInsetsPadding(WindowInsets.safeDrawing),
-          ) {
-            Spacer(Modifier.height(16.dp))
```

Comment:
This one is to bring "Insurances" title so it would be at the same place as other titles in the other bottom tabs (Forever etc.), and at the same place as "Insurance Details" title one screen deeper